<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Blackhole</class>
 <widget class="QMainWindow" name="Blackhole">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Blackhole</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnSelectGameDir">
        <property name="text">
         <string>Select game directory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSettings">
        <property name="text">
         <string>Settings</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAbout">
        <property name="text">
         <string>About</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="galaxyListView"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
